backend/database/seeders/DatabaseSeeder.php

```diff
@@ -77,9 +77,9 @@ public function run()
         $submission_file_seeder->run();
 
         $inlineCommentSeeder = new InlineCommentSeeder();
-        $inlineCommentSeeder->run(100, 1);
-        $inlineCommentSeeder->run(100, 0);
-        $inlineCommentSeeder->run(100, 10);
+        $inlineCommentSeeder->run(100, ['replies' => 1, 'highlight' => [30, 80]]);
```

Comment:
Seeing a mixture of integer and array in the parameters makes me think this can be refactored. I tend to think specifying the submission ID is largely useless and should be removed as a parameter. If necessary, then it should be included as part of the array. What do you think?

Reply:
The issue I see is there isn't a default for submissionId.  It has to be provided. Where as what's in the array has sensible defaults and can be optional.  Does that sound reasonable ?

Reply:
But there's only one seeded submission. This parameter doesn't provide any real benefit currently. For example, `SubmissionFileSeeder` just statically declares the ID. 

If you still believe the submission ID should be a parameter, it's possible to make the ID a required key in the array while having the other keys optional. 

I'm not saying you have to rewrite this. It's just a suggestion.